optaplanner-core/src/main/java/org/optaplanner/core/impl/solver/XStreamXmlSolverFactory.java

```diff
@@ -49,6 +49,8 @@
      */
     public static XStream buildXStream() {
         XStream xStream = new XStream();
+        String[] voidDeny = {"void.class", "Void.class"};
```

Comment:
Since this is a temporary workaround, we want to add a comment to make it easy to remove later, something like:
`
// TODO Workaround: remove when upgrading to XStream x.y.z or greater`